### DISCUSSION TITLE
Casaos Can be installed on a mounted disk?

### DISCUSSION THREAD

**Discussion by KoWming** [IceWhaleTech/CasaOS#336] [Q&A] [answered]
[FAILED] requires atleast 5GB disk space (Disk space on / is 3GB).

**Comment by jackwei1985** [comment, marked as answer]
脚本设定安装至少要5GB空间，可以把脚本下载到本地，然后修改这个代码 readonly MINIMUM_DISK_SIZE_GB="5" 改成3，在本地执行脚本安装。

**Reply by KoWming** [reply]
> 脚本设定安装至少要5GB空间，可以把脚本下载到本地，然后修改这个代码 readonly MINIMUM_DISK_SIZE_GB=“5” 改成3，在本地执行脚本安装。

好的，感谢帮助，我正在进行安装。

**Reply by EDmondqin** [reply]
但是我的主内存已经不足1G了，安装casa os要占多少内存？
